test_libs/pyspec/eth2spec/utils/minimal_ssz.py

```diff
@@ -6,6 +6,8 @@
 BYTES_PER_LENGTH_OFFSET = 4
 ZERO_CHUNK = b'\x00' * BYTES_PER_CHUNK
 
+cached_typedefs = {}
```

Comment:
```suggestion
```

Reply:
I think you don't need cached_typedefs anymore as you got rid of SSZTypeExtension